present.md

```diff
@@ -1,6 +1,8 @@
-This section focuses on more current developments in Faker JS (hereafter referred to as simply Faker).
+This section focuses on current developments in Faker JS (hereafter referred to as simply Faker).
 
-Let us start by looking at version 6 of Faker, which is demarked by the project being rewritten in typescript, adding documentation, setting up a CI/CD pipeline, and beginning to make major bug fixes regarding the implementation.
+The present version of Faker differs from the past Faker in several ways. Namely a dedicated source and documentation folder, as well as several key integrations that allow Faker to be automated. The current documentation is of much higher quality as well. Faker currently has a much more organized folder structure and many more configuration files that enable it to be automated efficiently.
+
+Let us move on to the most recent versions of Faker, starting with version 6, which is demarked by the project being rewritten in typescript, adding documentation, setting up a CI/CD pipeline, and beginning to make major bug fixes regarding the implementation.
```

Comment:
More recent versions of Faker starting at version 6 , demarked by being rewritten in typescript, has new features such as better documentation, setting up a CI/CD pipeline, and an increased effort to fix major bugs within the project